src/backend/funmooc/settings.py

```diff
@@ -206,6 +231,9 @@ class Base(DRFMixin, RichieCoursesConfigurationMixin, Configuration):
         "parler",
         "rest_framework",
         "storages",
+        "social_django",
+        # Fun mooc
+        "funmooc.auth.apps.AuthConfig",
```

Comment:
What is the point of declaring this as an app?

Reply:
I also think it's useless. I've left this from the first implementation.

Reply:
After thought: this app should be declared so that the template loader can find the `login.html` registration template.